metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-security-env.xml

```diff
@@ -121,6 +121,7 @@
     <display-name>User password attribute</display-name>
     <description>LDAP attribute for the user password</description>
     <value-attributes>
+      <type>password</type>
```

Comment:
While here, I also set the LDAP password type appropriately.

Reply:
This isn't actually a password, it's the LDAP attribute that contains the password for a given user.  It doesn't need the password type.

The default is just the value "userPassword", because that's at least often where the value is stored.

Reply:
Oh okay, got it. Sorry, I mis-interpreted it. Let me revert this change.

Reply:
To add to this, check out https://github.com/apache/metron/blob/fefbb376fad0526116ed91035408bfeb72059d9c/metron-deployment/development/knox-demo-ldap.ldif#L55

The field is "userPassword", but the actual value is "admin-password". In an actual instance with hashed passwords it might be "userPassword:{SSHA}<hashed password>".

Reply:
Yup, makes sense.